Lib/unittest/case.py

```diff
@@ -338,6 +339,25 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
+def short_description_from_docstring(text):
+    """
+    Return the test case short description from a docstring.
+
+    Ths docstring is parsed by the standard `pydoc.splitdoc` function
```

Comment:
Docstrings of internal functions are not forbidden, they are just not necessary. Comments would be enough (see PEP 8).

In any case neither comment, nor docstring shouldn't just repeat the code. They should describe *what* function does, but not *how* it does this. Otherwise any change of the code will make a docstring or a comment incorrect.

Note also that the function name itself serves the documenting function.